osu.Game/Screens/Play/HUD/ArgonStrainGraph.cs

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Lines;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Beatmaps;
+using osu.Game.Configuration;
+using osu.Game.Localisation.HUD;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public partial class ArgonStrainGraph : SongProgress
+    {
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.ShowBackground))]
+        public BindableBool ShowBackground { get; } = new BindableBool(true);
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.EnableGradient), nameof(StrainGraphStrings.EnableGradientDescription))]
+        public BindableBool UseBackgroundGradient { get; } = new BindableBool(true);
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.UseAdditiveBlending), nameof(StrainGraphStrings.AdditiveBlendingDescription))]
+        public BindableBool UseAdditiveBlending { get; } = new BindableBool(true);
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.BackgroundColour))]
+        public BindableColour4 BackgroundColour { get; } = new BindableColour4(Color4Extensions.FromHex("#66CCFF"));
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.LineColour))]
+        public BindableColour4 LineColour { get; } = new BindableColour4(Color4.White);
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.HorizontalSpacing))]
+        public BindableInt HorizontalSpacing { get; } = new BindableInt(5)
+        {
+            MinValue = 3,
+            MaxValue = 10,
+        };
+
+        [SettingSource(typeof(StrainGraphStrings), nameof(StrainGraphStrings.VerticalSpacing))]
+        public BindableInt VerticalSpacing { get; } = new BindableInt(10)
+        {
+            MinValue = 5,
+            MaxValue = 20,
+        };
```

Comment:
these should not be settings, you should just be able to resize the graph vertically and horizontally using the drag handles